week-1/implement/max.js

```diff
@@ -0,0 +1,11 @@
+function maximum(arr){
+ if(arr.length === 0){
+    return -Infinity;
```

Comment:
It's more common to return `null` in this case.
Some extra reading if you want https://www.geeksforgeeks.org/undefined-vs-null-in-javascript/

Reply:
This link is more help full I read about Null and undefined Thank you